### ISSUE TITLE
Vagrant fails for "fedora/27-cloud-base"

### ISSUE THREAD

**Issue by gclough** [ANXS/postgresql#390]
When running Vagrant with "fedora/27-cloud-base", it fails:

```
fatal: [fedora27.local]: FAILED! => {
    "changed": false, 
    "module_stderr": "Shared connection to 192.168.88.27 closed.\r\n", 
    "module_stdout": "\r\n/bin/sh: /usr/bin/python: No such file or directory\r\n", 
    "msg": "MODULE FAILURE", 
    "rc": 0
}
```

**Comment by gclough** [comment]
If you manually install python after the machine is provisioned, then it works... but this step is not required for any other the other machine types.

**Comment by jlozadad** [comment]
yeah I believed I noted that somewhere around. Fedora does not include python on the vagrant image.

**Comment by gclough** [comment]
OK, sure.  In that case, I've left it commented out of the Vagrantfile in #392 .  If there is a way to add python to the VM after it's provisioned, then that would be a good addition, but if not, then we'll just have to leave it out of Vagrant.

**Comment by gclough** [comment]
Fixed by #393
